src/custom/pages/Faq/index.tsx

```diff
@@ -344,7 +345,7 @@ export default function Faq() {
           </p>
           <p>
             Be cautious, some people may create fake COW tokens, that are not affiliated with this project. Please note
-            that any token listed in any AMM is <strong>NOT</strong> associated with this project in any way, shape or
+            that any token listed in any AMM is <strong>NOT</strong> associated with this project in any way, shape, or
```

Comment:
in or on? "on any AMM"

Reply:
According to grammerly, in is the correct one.